tf_big/cc/kernels/big_kernels.cc

```diff
@@ -57,13 +58,14 @@ class BigExportOp : public OpKernel {
 
   void Compute(OpKernelContext* ctx) override {
     const BigTensor* val = nullptr;
+    TensorShape input_shape = ctx->input(0).shape();
```

Comment:
probably not the most elegant to have this here to infer the output shape. But even though the input is a scalar, after running `GetBigTensor(ctx, 0, &val),  we get`val->rows() = 1` and `val->cols() = 1` so couldn't use this to set output shape.